import problem on @h5web/h5wasm components when using NextJS

### Describe the bug

Hello people. The description is that i tried to create a simple NextJs using @h5web/h5wasm. But for some reason, this package is having a import the import error " Module not found: Can't resolve './' ". Can you guys help me out with this task ?

### To Reproduce

1. install the project with NextJs basic stac : npx create-next-app@latest
2. Use the following options
```
✔ What is your project named? … test-h5web
✔ Would you like to use TypeScript? … No / Yes
✔ Would you like to use ESLint? … No / Yes
✔ Would you like to use Tailwind CSS? … No / Yes
✔ Would you like to use `src/` directory? … No / Yes
✔ Would you like to use App Router? (recommended) … No / Yes
✔ Would you like to customize the default import alias (@/*)? … No / Yes
Creating a new Next.js app in ~/test-h5web

Using npm.

Initializing project with template: app-tw 


Installing dependencies:
- react
- react-dom
- next

Installing devDependencies:
- typescript
- @types/node
- @types/react
- @types/react-dom
- autoprefixer
- postcss
- tailwindcss


added 117 packages, and audited 118 packages in 9s

found 0 vulnerabilities
Initialized a git repository.

Success! Created bla at ~/test-h5web
```
3. use 
```
yarn add @h5web/app @h5web/h5wasm
```
4. Then, replace ```src/app/page.tsx``` with the code
```
"use client"
import '@h5web/app/dist/styles.css';

import React, { useState } from 'react';
import type { ChangeEvent } from 'react';
import { App } from '@h5web/app';
import { H5WasmProvider } from '@h5web/h5wasm';

interface H5File {
  filename: string;
  buffer: ArrayBuffer;
}

function MyApp() {
  const [h5File, setH5File] = useState<H5File>();

  function handleFileInputChange(evt: ChangeEvent<HTMLInputElement>) {
    const file = evt.target.files?.[0];
    if (!file) {
      return;
    }

    /* `H5WasmProvider` expects an HDF5 file in the form of an `ArrayBuffer`.
     * The way you get this buffer is up to you. Here, we show a simple file picker
     * and use the FileReader API to process the selected file. */
    const reader = new FileReader();
    reader.addEventListener('load', () => {
      setH5File({
        filename: file.name,
        buffer: reader.result as ArrayBuffer,
      });
    });

    reader.readAsArrayBuffer(file);
  }

  if (!h5File) {
    return (
      <input
        aria-label="Pick HDF5 file"
        type="file"
        accept=".h5"
        onChange={handleFileInputChange}
      />
    );
  }

  return (
    <div style={{ height: '100vh' }}>
      <H5WasmProvider {...h5File}>
        <App />
      </H5WasmProvider>
    </div>
  );
}

export default MyApp;
   ```
### Expected behaviour
This's a basic "hello world " using the example on [@h5web/h5wasm](https://www.npmjs.com/package/@h5web/h5wasm). IT's expected to display a application like the examples on the h5web repository (An example is in this [link](https://h5web.panosc.eu/h5wasm))

### Screenshots
![image](https://github.com/silx-kit/h5web/assets/33372797/b09ca7a6-c78a-44f2-ad6f-1f52678adaf3)



### Context

- OS: Ubuntu 22.04
- Browser: Chrome
- Version: Version 116.0.5845.140 (Official Build) (64-bit)
- H5Web context: demo, @h5web/lib, @h5web/app, @h5web/h5wasm

--- a/packages/h5wasm/package.json
+++ b/packages/h5wasm/package.json
@@ -45,7 +45,7 @@
     }
   },
   "dependencies": {
-    "h5wasm": "0.6.9",
+    "h5wasm": "0.7.0",
     "nanoid": "5.0.1"
   },
   "devDependencies": {

